src/octree/mod.rs

```diff
@@ -417,19 +418,20 @@ impl Octree {
         beam: &'a OrientedBeam,
     ) -> FilteredPointsIterator<'a> {
         let mut node_ids = VecDeque::new();
-        let mut open_list = vec![Node::root_with_bounding_cube(Cube::bounding(
+        let mut open_list = VecDeque::new();
+        open_list.push_back(Node::root_with_bounding_cube(Cube::bounding(
```

Comment:
Could we also use vec![...].into() here like above?

Reply:
Yes, it doesn't make much difference though in terms of readability... I adjusted it.